package/CreateChocoPackage.ps1

```diff
@@ -0,0 +1,10 @@
+$content=(Get-Content -path tools\chocolateyinstall.ps1 -Raw)
```

Comment:
Is it possible to add a short description in the comments at the top of this script, with some instruction of usage?

Reply:
Added info